app/controllers/system_settings_controller.rb

```diff
@@ -10,9 +11,11 @@ def show
 
   def new
     @system_setting = SystemSetting.new
+    @exchange_types = SystemSetting::EXCHANGE_TYPES
```

Comment:
EXCHANGE_TYPES are things like peer_to_peer, dispatch_moderated, etc
TODO - work through state machine for lifecycle methods to get accurate EXCHANGE_TYPES list